taskcluster/fenix_taskgraph/target_tasks.py

```diff
@@ -32,19 +32,28 @@ def target_tasks_nightly(full_task_graph, parameters, graph_config):
     def filter(task, parameters):
         return (
             task.attributes.get("nightly", False) and
-            not _filter_fennec_nightly(task, parameters)
+            not _filter_fennec("nightly", task, parameters)
         )
 
     return [l for l, t in full_task_graph.tasks.iteritems() if filter(t, parameters)]
 
-def _filter_fennec_nightly(task, parameters):
-    return task.attributes.get("build-type", "") == "fennec-nightly"
+
+def _filter_fennec(fennec_type, task, parameters):
```

Comment:
++ for this refactoring, we can re-use this in `beta` and later on in `release` as well.